tests/performance-tests/src/configuration/steadyRequestRateProfile.ts

```diff
@@ -0,0 +1,70 @@
+import { Options } from 'k6/options';
+import { mergeObjects, parseIntOptional } from '../utils/utils';
+
+interface Config {
+  // Name of the scenario.
+  scenario?: string;
+  // Total duration of the main stage of the test, providing a steady rate of requests.
+  mainStageDurationMinutes: number;
+  // Requests per second to generate during the main stage.
+  steadyRequestRatePerSecond: number;
+  // Duration of a short final stage to allow any long-running requests to complete.
+  cooldownStageDurationMinutes: number;
+}
+
+const overrides: Partial<Config> = {
+  steadyRequestRatePerSecond: parseIntOptional(__ENV.RPS),
+  mainStageDurationMinutes: parseIntOptional(
+    __ENV.MAIN_TEST_STAGE_DURATION_MINS,
+  ),
+  cooldownStageDurationMinutes: parseIntOptional(
+    __ENV.MAIN_TEST_STAGE_DURATION_MINS,
+  ),
+};
+
+export default function steadyRequestRateProfile({
+  scenario,
+  ...defaultConfig
+}: Config & { scenario?: string }): Options {
+  const {
+    steadyRequestRatePerSecond,
+    mainStageDurationMinutes,
+    cooldownStageDurationMinutes,
+  } = mergeObjects(defaultConfig, overrides);
+
+  return {
+    scenarios: {
+      [scenario ?? 'steady_rate']: {
```

Comment:
Could define this default in the function parameter?

Reply:
:+1:

Reply:
Done ta